README.md

```diff
@@ -1,2 +1,47 @@
-# restaurant-page
-Js Project
+# Restaurant-Page
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+
+A simple restaurant website.
+
+This website was built using just JavaScript to generate all of the content on the page.
+
+Additional description about the project and its features.
+
+## Built With
+
+![JAVASCRIPT](https://img.shields.io/badge/javascript%20-%23323330.svg?&style=for-the-badge&logo=javascript&logoColor=%23F7DF1E")
+![CSS](https://img.shields.io/badge/css3%20-%231572B6.svg?&style=for-the-badge&logo=css3&logoColor=white)
+![BOOTSTRAP](https://img.shields.io/badge/bootstrap%20-%23563D7C.svg?&style=for-the-badge&logo=bootstrap&logoColor=white) 
+
+
```

Comment:
### [OPTIONAL]
- [ ] It will be great to add a live preview of the project in your readme file. This will make your project even more professional.